spec/factories.rb

```diff
@@ -1,2 +1,6 @@
 FactoryBot.define do
+  factory :expense do
+    sequence(:name) { |n| "name#{n}" }
+    sequence(:frequency) { |n| "frequency#{n}" }
```

Comment:
Since frequency is not unique, a suggestion to use a value that is closer to real life